### ISSUE TITLE
NTR: Antisperm antibodies present (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8730]
New term request Antisperm antibodies present (elthat_100926163115):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000188
- label: Antisperm antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:31527552
- def: The presence of autoantibodies (immunoglobulins) in the serum that react against sperm antigens.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8730
- synonyms:
